client/src/app/site/full-search/full-search.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { FullSearchComponent } from './full-search.component';
```

Comment:
need to imort the e2e imports module. check other `*.spec.ts` files to get the idea